src/Components/Notification.tsx

```diff
@@ -0,0 +1,22 @@
+interface Props {
+  message: string,
+  handleClose: () => void,
+}
+
+export const Notification: React.FC<Props> = ({ message, handleClose }) => {
+  return (
+    <div className={`notification is-danger is-light has-text-weight-normal ${!message && 'hidden'}`}>
```

Comment:
It's better to use a ternary operator for conditional rendering instead of checking for a falsy value